components/track/TrackChangeLocale.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { LanguageEnum } from "@bcc-code/bmm-sdk-fetch";
+import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/vue";
+
+const expanded = ref(false);
+const { currentTrack } = useNuxtApp().$mediaPlayer;
+const { t } = useI18n();
+
+const changeLanguage = (lang: LanguageEnum) => {
+  expanded.value = false;
+  contentLanguageStore().selectedLanguage = lang;
+};
+
+const getSortedLanguages = () => {
+  const contentLangs = contentLanguageStore()
+    .contentLanguages as LanguageEnum[];
+  const tempLangs = currentTrack?.value?.languages || [];
+  const sortedLanguages = contentLangs.filter((lang) =>
+    tempLangs.includes(lang),
+  );
+  tempLangs.forEach((lang) => {
+    if (sortedLanguages.includes(lang)) {
+      return;
+    }
+    sortedLanguages.push(lang);
+  });
+
+  return sortedLanguages;
+};
+const firstFiveLanguages = () => getSortedLanguages().slice(0, 5);
+const allLanguages = () => getSortedLanguages();
+</script>
+<template>
+  <Menu
+    as="div"
+    class="relative text-left flex flex-col justify-center"
+    @click.stop
+  >
+    <MenuButton
+      as="button"
+      :aria-label="t('track.a11y.options')"
+      class="rounded-full p-1 hover:bg-background-2 hover:text-label-1"
+    >
+      {{
+        currentTrack?.language
+          ? getLocalizedLanguageName(currentTrack.language)
```

Comment:
getLocalizedLanguageName() is awesome. I can probably use that in #310. Thanks for finding it.

Reply:
the credit should go to @SimonSimCity - he added it as placeholder for the language selection.